src/core/QuoridorPlayer.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace QuoridorEngine.src.quoridor
+﻿namespace QuoridorEngine.src.core
```

Comment:
Remove .src from namespace. Unnecessary and ties namespace to folder structure which might change later

Also use .Core with a capital C to follow naming conventions